### PR TITLE
add sftpgo

### PR DESCRIPTION
**Pull request by 9uanhuo**

### CODE REVIEW

**Review by hoilc — APPROVED** [review]

File: bucket/sftpgo.json [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+        "SFTPGO_HTTPD__BACKUPS_PATH": "$dir\\backups"
+    },
+    "persist": [
+        "config",
+        "backups"
+    ],
+    "post_install": [
+        "if (! (Test-Path \"$dir\\config\\sftpgo.json\")) { Copy-Item -Path \"$dir\\sftpgo.json\" -Destination \"$dir\\config\\sftpgo.json\" }",
+        "if (! (Test-Path \"$dir\\config\\sftpgo.db\")) { Copy-Item -Path \"$dir\\sftpgo.json\" -Destination \"$dir\\config\\sftpgo.db\" }",
+        "Remove-Item \"$dir\\sftpgo.json\", \"$dir\\sftpgo.json\" -Force"
```
Comment (9uanhuo) [inline bucket/sftpgo.json]: Should be `"Remove-Item \"$dir\\sftpgo.db\", \"$dir\\sftpgo.json\" -Force"`

**Review by 9uanhuo — COMMENTED** [review]
lol

File: bucket/sftpgo.json (hunk 1 above)
Comment (hoilc) [reply bucket/sftpgo.json]: thx. I could only work on macOS during working hours😂

**Pull request merged**